util/cron/common-ofi.bash

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+#
+# Configure environment for CHPL_COMM=ofi testing.
+
+CWD=$(cd $(dirname ${BASH_SOURCE[0]}) ; pwd)
+
+source $CWD/common.bash
+source $CWD/common-oversubscribed.bash
+
+export CHPL_COMM=ofi
+
+# Select a launcher and out-of-band support.  If we're on a Cray XC
+# system this will just fall out automatically.  On a slurm-based system
+# where slurm was configured with the PMIx server-side plugin and we can
+# find a PMIx client-side library, then we can use slurm-srun directly,
+# along with the comm=ofi slurm-pmi2 out-of-band support.  This is seen
+# on Cray CS systems, for example.  Otherwise, our only option is the
+# stopgap mpirun4ofi launcher, in which case we also need MPI and the
+# MPI-based out-of-band support.
+lch=$($CHPL_HOME/util/chplenv/chpl_launcher.py)
+lchOK=
+
+if [ $($CHPL_HOME/util/chplenv/chpl_platform.py --target) == cray-x* ] ; then
+  lchOK=y
+else if [ "$lch" == slurm-srun ] && \
+        srun --mpi=list 2>&1 | grep -q pmix && \
+        module avail pmix 2>&1 | grep -q pmix ; then
+  module load pmix
+  export CHPL_COMM_OFI_OOB=slurm-pmi2
+  export SLURM_MPI_TYPE=pmix
+  lchOK=y
+fi
+
+if [ -z "$lch" ] ; then
+  export CHPL_LAUNCHER=mpirun4ofi
+
+  # Load OpenMPI environment module and confirm it has loaded
+  echo >&2 module load gnu-openmpi
+  module load gnu-openmpi
+
+  set -x
+  : confirm mpi module is loaded
```

Comment:
Tiny nit-pick: I prefer to use '#' for comments, as ':' has some other (weird) uses in bash. Or maybe this is one of those? Up to you.

Reply:
I removed it -- it was there because I stole it from common-mpicc, but it's gone from there now, too.  And it was superfluous, given the comment right above it.